bittensor/extrinsics/prometheus.py

```diff
@@ -74,20 +73,20 @@ def prometheus_extrinsic(
     else:
         external_ip = ip
 
-    call_params: "bittensor.PrometheusServeCallParams" = {
+    call_params: "PrometheusServeCallParams" = {
         "version": bittensor.__version_as_int__,
         "ip": net.ip_to_int(external_ip),
         "port": port,
         "ip_type": net.ip_version(external_ip),
     }
 
     with bittensor.__console__.status(":satellite: Checking Prometheus..."):
-        neuron = subtensor.get_neuron_for_pubkey_and_subnet(
+        neuron = await subtensor.get_neuron_for_pubkey_and_subnet(
             wallet.hotkey.ss58_address, netuid=netuid
         )
         neuron_up_to_date = not neuron.is_null and call_params == {
             "version": neuron.prometheus_info.version,
-            "ip": net.ip_to_int(neuron.prometheus_info.ip),
+            "ip": neuron.prometheus_info.ip,
```

Comment:
does neuron.prometheus_info.ip still return an 'int'?

Reply:
I forgot to replace this back. done